lxd/identity/cache.go

```diff
@@ -186,3 +186,21 @@ func (c *Cache) GetByOIDCSubject(subject string) (*CacheEntry, error) {
 
 	return nil, api.StatusErrorf(http.StatusNotFound, "Identity with OIDC subject %q not found", subject)
 }
+
+// GetIdentityProviderGroupMapping returns the auth groups that the given identity provider group maps to or an
+// api.StatusError with http.StatusNotFound.
+func (c *Cache) GetIdentityProviderGroupMapping(idpGroup string) ([]string, error) {
```

Comment:
is this related to or dependent on the openfga driver?

Reply:
Yes, it's used in the driver to get the groups that the IDP groups map to and add them as contextual tuples.

I can add it in a separate commit but I'll need to add a `//nolint` as it will be unused.

Reply:
OK keep it as is then